samples/bluetooth/hci_rpmsg/prj.conf

```diff
@@ -2,17 +2,22 @@ CONFIG_IPC_SERVICE=y
 CONFIG_MBOX=y
 
 CONFIG_HEAP_MEM_POOL_SIZE=8192
-
-CONFIG_MAIN_STACK_SIZE=512
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
 
 CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
 CONFIG_BT_HCI_RAW_RESERVE=1
-CONFIG_BT_MAX_CONN=16
+CONFIG_BT_MAX_CONN=4
```

Comment:
nRF53x has only 64KB RAM, hence will not override the MAIN_STACK_SIZE in case in the future the Kconfig default modify it across the repo. MAX_CONN was reduced so that direction finding can use required RAM for generating reports.